src/__tests__/store/reducers/apiResponse.reducer.test.js

```diff
@@ -75,12 +76,14 @@ describe('apiResponseReducer', () => {
         const apiName = getRandomString();
         const message = getRandomString();
         const onConfirm = getRandomString();
+        const shouldRedirect = getRandomString();
```

Comment:
why is it initialized as a string?, the name kind a implies a boolean val

Reply:
in this particular case, I'm checking the existence and assigning of the property, so basically it doesn't matter. But agree, it's definitely better/consistent to work with proper type.